python/ray/_private/services.py

```diff
@@ -1929,3 +1929,22 @@ def start_ray_client_server(redis_address,
         stderr_file=stderr_file,
         fate_share=fate_share)
     return process_info
+
+
+def get_object_locations(obj_refs, timeout_ms=-1):
+    """Get object locations.
```

Comment:
Can you update the docstring to include more details here? E.g., example output + when None or error is included in the output, and etc..

Reply:
And maybe moving the function to `internal_api.py`